app/_layouts/extension.html

```diff
@@ -216,13 +304,47 @@
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the Route that this plugin configuration
+  will target.
+
 {% else %}
 {% navtabs %}
 {% navtab Admin API %}
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the Route that this plugin configuration
+  will target.
+
 {% endnavtab %}
+{% unless page.params.manager_examples == false %}
+{% navtab Kong Manager %}
+Configure this plugin on a Route:
+
+1. In Kong Manager, select the workspace.
+2. From the Dashboard, select **Routes** in the left navigation.
+3. Click **View** for the Route row.
+4. Scroll down to plugins and click **Add Plugin**.
+5. Find and select the **{{ page.name }}** plugin.
+
+    <div class="alert alert-ee blue">
+    <strong>Note:</strong> If the plugin is greyed out, then it is not available
+    for your product tier. See <a href="">Kong Gateway tiers</a>.
+    </div>
+
+6. If the option is available, set the plugin scope to **Scoped**.
```

Comment:
Any more information about scoped?

Reply:
not sure what you mean here?

Reply:
The definition is in the UI, I don't think we need to duplicate the info here. UI currently says:
```
Scoped
Specific Services and/or Routes in this workspace
```